keyboards/ergodash/mini/keymaps/ToyoshimaHidenori/keymap.c

```diff
@@ -0,0 +1,141 @@
+/* Copyright 2020 Toyoshima Hidenori <t@toyo.dev>
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>
+*/
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```

Comment:
This isn't actually needed.

```suggestion
```

Reply:
Thank you for your code review!
I have fixed code according your suggestions in e5890bd.
Thank you.